_posts/2024-01-29-strimzicon2024-announcement.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title:  "Welcome to the StrimziCon 2024!"
+date: 2024-01-26
+author: paolo_patierno
+---
+
+We are very pleased to announce the first StrimziCon coming this year!
```

Comment:
```suggestion
We are very pleased to announce the inaugural StrimziCon, scheduled later this year!
```

Reply:
Why is "inaugural" better than "first"?

Reply:
I thought inaugural sounded more significant and like something is being established. Just a suggestion